api/public_link_manager_owncloud/public_link_manager_owncloud.go

```diff
@@ -11,20 +11,21 @@ import (
 	"github.com/cernbox/revaold/api"
 
 	"database/sql"
+	"math/rand"
```

Comment:
Should use `crypto/rand` instead of `math/rand`: now public links generated through revaold are predictable (and then the seeding with the time that reva was started can also be removed)